docs/src/tutorials/creating_geometry.md

```diff
@@ -7,15 +7,17 @@ import GeoFormatTypes as GFT
 import CoordinateTransformations
 import Proj
 using CairoMakie
+using GeoMakie
```

Comment:
Note GeoMakie needs to be loaded for any plotting to work, so I've moved all needed packages to the start

Reply:
Ah, that's because GeoInterfaceMakie needs to be loaded as well.  GeoInterface can't actually do that since it's a dependency of Makie, any other package geometry would not require this.